### PR TITLE
COT-917: Add removed judge and bookable for identified fee-paid judge

### PR DESCRIPTION
**Pull request by ben-james-hmcts**
Revert "COT-912 ST_CIC remove booked judge role assignment"
Reverts hmcts/am-org-role-mapping-service#2145

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java [hunk 1]
```diff
@@ -49,12 +49,16 @@ class DroolStcicJudicialRoleMappingTest extends DroolBase {
         expectedRoleNameWorkTypesMap.put("fee-paid-financial", null);
     }
 
-    static void assertCommonRoleAssignmentAttributes(RoleAssignment r) {
+    static void assertCommonRoleAssignmentAttributes(RoleAssignment r, String office, List<String> ticketCodes) {
```
Comment (abul-syed) [inline src/test/java/uk/gov/hmcts/reform/orgrolemapping/domain/service/DroolStcicJudicialRoleMappingTest.java:52]: should we can omit `office` & `ticketCodes` from input param as not used in test?

**Review by abul-syed — CHANGES_REQUESTED** [review]
nice PR, minor feedback

**Review by abul-syed — APPROVED** [review]
lgtm

**Review by mattnayler — APPROVED** [review]
LGTM

**Pull request merged**